internal/gps/pkgtree/pkgtree.go

```diff
@@ -445,6 +450,15 @@ func (t PackageTree) ToReachMap(main, tests, backprop bool, ignore map[string]bo
 		ignore = make(map[string]bool)
 	}
 
+	// Create a radix tree for ignore prefixes
+	xt := radix.New()
```

Comment:
this allocs because it returns a pointer, so let's defer creating the trie until the last possible moment - _if_ we see something with the expected suffix.